lib/test/revision.go

```diff
@@ -139,3 +139,10 @@ func RevisionListWithService(r *KnRunResultCollector, serviceNames ...string) {
 		}
 	}
 }
+
+// RevisionDeleteWithAllOption verifies deleting all revision in sync mode
+func RevisionDeleteWithAllOption(r *KnRunResultCollector, revName string) {
+	out := r.KnTest().Kn().Run("revision", "delete", "--all")
+	assert.Check(r.T(), util.ContainsAll(out.Stdout, "Revision", revName, "deleted", "namespace", r.KnTest().Kn().Namespace()))
+	r.AssertNoError(out)
+}
```

Comment:
either this should take multiple revision names to verify or no revisions names at all (could then just verify without revision names).